### PR TITLE
Preserve empty lines in CSS/SCSS paren groups

### PR DESCRIPTION
**Pull request by duailibe**
## Description

Related to #12053, will improve the comment placement from the example in another PR

## Checklist

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Review by sosukesuzuki — COMMENTED** [review]
Can you add tests for multi empty lines?

**Prettier pr-12210**
[Playground link](https://deploy-preview-12210--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEASSAbCBXATgZyQAIAKAHSiKqLJHgA8ZbjUMBLAcwAsYBaARgAMggDQVqNEACMAhmADWHXDigATXgAdcbALYzcAT2ZFUq-fN4BmYWKgV7dx+Oq1ZCpSvX44kNfqPIJma4FgBMNs5UrnKKythqvPC4MGz+xqbmAsIUAJQA3CAiIBAaKdD4yKD6ygDuAAr6CBUoMhg1MgYVRVK4MXAwAMoyOnAAMmxQcMgAZq3e3b0K-QMachMcyDC42HBFcDpScKqqR6MyUBzYMhxwAGIQuHowKRfIIDLYMBCFIDw6GAB1LhseD4VZgOADJogtgANxBBjeYHwXRAE28yTqvQ4ehmc12IAAVvh6AN1hg4ABFbAQeB4jDzECrAhwXBvfDI1FaCYwAFsVQwLjIAAcoiZym8AN6GjeWjgGNhUyKAEcafAsSVmu98LxJkcjj9cHBVWwjVjrrikLMGQTvDo2JttrbyVS1VMrfiijAZFI+QKhUhQl7emx2BcAMIQHSWkDygCsP2w3gAKj7mtbGbCdgBJNQIQZgbSlACCagGMAMFPp3gAvjWgA)

```sh
--parser scss
```

**Input:**

```scss
$colours: (
    "text": $light-100,
    "background-primary": $dark-300,






    "background-secondary": $dark-200,
    "background-tertiary": $dark-100
);
```

**Output:**

```scss
$colours: (
  "text": $light-100,
  "background-primary": $dark-300,

  "background-secondary": $dark-200,
  "background-tertiary": $dark-100,
);

```

File: changelog_unreleased/css/12210.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#### Preserve empty line in CSS paren groups (#12210 by @duailibe)
+
+<!-- Optional description if it makes sense. -->
+
+<!-- prettier-ignore -->
+```css
+// Input
```
Comment (sosukesuzuki) [inline changelog_unreleased/css/12210.md:7]:
```suggestion
/* Input */
```

File: changelog_unreleased/css/12210.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+$colours: (
+    "text": $light-100,
+
+    "background-primary": $dark-300,
+    "background-secondary": $dark-200,
+    "background-tertiary": $dark-100
+);
+
+// Prettier stable
```
Comment (sosukesuzuki) [inline changelog_unreleased/css/12210.md:16]:
```suggestion
/* Prettier stable */
```

File: changelog_unreleased/css/12210.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+$colours: (
+  "text": $light-100,
+  "background-primary": $dark-300,
+  "background-secondary": $dark-200,
+  "background-tertiary": $dark-100
+);
+
+// Prettier main
```
Comment (sosukesuzuki) [inline changelog_unreleased/css/12210.md:24]:
```suggestion
/* Prettier main */
```

File: changelog_unreleased/css/12210.md (hunk 1 above)
Comment (duailibe) [reply changelog_unreleased/css/12210.md]: Oops, nice catch

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**